objects/Camera.js

```diff
@@ -14,7 +14,6 @@ class Camera {
     }
 
     static fromXML(XMLObject) {
-        // TODO: replace ip
```

Comment:
I will remove this in the next sprint, for now it is necessary for compatibility with the scripting applixation